types/node-vault-client/.eslintrc.json

```diff
@@ -0,0 +1,8 @@
+{
+    "rules": {
+        "@definitelytyped/dt-header": "off",
+        "@definitelytyped/no-declare-current-package": "off",
+        "@definitelytyped/no-single-declare-module": "off",
+        "@typescript-eslint/ban-types": "off"
```

Comment:
Please delete this file and fix the issues dtslint mentions; silencing these errors isn't the right way forward.

Reply:
Apologies @jakebailey - copied from `node-validator`. This has now been removed and linting errors fixed.